bentoml/service/__init__.py

```diff
@@ -66,11 +66,23 @@ def validate_inference_api_name(api_name: str):
         )
 
 
+def validate_inference_api_route(route: str):
+    if re.findall(r"[?#]+|^(//)|^:", route):
```

Comment:
Let's add documentation for the regex.

Reply:
Should we also invalidate route input like `abc/123/` with trailing '/'?

Reply:
The route with a trailing "/" also works.